### PR TITLE
JBPEAP-22293 new quickstart to show access for RHDG caches

### PR DESCRIPTION
**Pull request by wfink**
https://issues.redhat.com/browse/JBEAP-22293

Adding an example how to access the configured Infinispan cache

### CODE REVIEW

File: clustered-cache/src/main/java/org/jboss/as/quickstarts/datagrid/eap/subsystem/web/JsfController.java [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+//   @Resource(lookup = "java:jboss/infinispan/container/jdg-container")
+//   CacheContainer container;   // container could be injected if needed
+
+  /**
+   * The direct lookup like
+   *
+   * @Resource(lookup = "java:jboss/infinispan/cache/jdg-container/EAPcache")
+   *
+   * is not working because of https://issues.redhat.com/browse/WFLY-11817.
```
Comment (pferraro) [inline clustered-cache/src/main/java/org/jboss/as/quickstarts/datagrid/eap/subsystem/web/JsfController.java]: I would instead link to the corresponding JBEAP jira:
https://issues.redhat.com/browse/JBEAP-22290

Comment (wfink) [reply clustered-cache/src/main/java/org/jboss/as/quickstarts/datagrid/eap/subsystem/web/JsfController.java]: done

**Comment by emmartins** [comment]
this should instead be contributed to wildfly/quickstart , and have a WFLY JIRA

**Comment by wfink** [comment]
> this should instead be contributed to wildfly/quickstart , and have a WFLY JIRA

more important for the EAP release as it is the first time this is supported.
With WildFly/Infinispan this could be done no matter which release

**Comment by wfink** [comment]
Rebased on top of 7.4.x and tested

**Comment by emmartins** [comment]
@wfink this is not a dev branch or repo, it's just where we publish released EAP quickstart sources, for OpenShift usage mainly. Any development must be done upstream so I suggest that if you still want to propose this then clone/move the JBEAP to WFLY and let's discuss it.

**Pull request closed without merging**